client/src/components/SubmissionForm.jsx

```diff
@@ -54,8 +54,27 @@ export default function SubmissionForm() {
         categories: '',
       }}
       validationSchema={validationSchema}
-      onSubmit={(values) => {
-        alert(JSON.stringify(values, null, 2));
+      onSubmit={async (values) => {
+
+        const submissionModel = {
+          title: values.title,
+          description: values.description,
+          technologies: values.technologies.split(','),
+          urls: values.links.map((link) => ({
+              label: link.label,
+              url: link.url,
+            }))
+        }
+
+        const postRequest = {
+          method: 'POST',
+          headers: { 'Content-Type': 'application/json' },
+          body: JSON.stringify(submissionModel),
+        };
+
+        const response = await fetch('http://localhost:6000/api/submissions', postRequest);
```

Comment:
Isn't the server's port at 5000? Why is it at 6000? Is that the reason why that error keeps popping up?